### DISCUSSION TITLE
How can I use the -tranform option with `GoTagAdd` cmd?

### DISCUSSION THREAD

**Discussion by nomorechokedboy** [olexsmir/gopher.nvim#28] [Q&A] [answered]
Currently, I am using gopher `GoTagAdd` cmd to auto generate tags for my struct. But the default transform is snake_case, which I would like to change into camelCase. I discovered that `gomodifytags` has an option for my problem which is -transform. But when I use `GoTagAdd json -transform camelCase`, `gopher` can't run the command. It would be nice if the readme have an example of how to pass -tranform camelCase from `GoTagAdd` cmd @olexsmir

**Comment by HichOoMa** [comment, marked as answer]
still without no answer ?

**Reply by olexsmir** [reply]
that's already merged, you can set `-transform` value in `setup()`

```lua
require("gopher").setup {
  gotag = {
    transform = "snakecase",
  },
}
```
